src/site/xdoc/server/config-spring-jpa-postgres.xml

```diff
@@ -0,0 +1,102 @@
+      <subsection name="Step 1: Create database on Postgres">
+
+        <p>You need to install Postgres or run Postgres with Docker</p>
+
+        <p>Sample to install like this</p>
+        <source>
+          sudo apt-get update && sudo apt-get install postgresql-9.4 postgresql-client-9.4
+        </source>
+        <p>Sample to pull and launch Postgres</p>
+        <source>
+          docker pull library/postgres
+        </source>
+        <p>Postgres does not create database on the fly so you need to create the database manually. Here is the command to perform this action: </p>
+
+        <source>psql -U postgres -tc "SELECT 1 FROM pg_database WHERE datname = '<strong><italic>database_name</italic></strong>'" | grep -q 1 || psql -U postgres -c "CREATE DATABASE <strong><italic>database_name</italic></strong>"</source>
+
+        <dl>
+          <dt><strong>database_name</strong></dt>
+          <dd>The name of database</dd>
+        </dl>
+
+      </subsection>
+
+      <subsection name="Step 2: Point to the database in configuration file">
+        <p>Change the content of <italic>james-database.properties</italic> like this. Point to the 'database_name' as above step</p>
+        <source>
+          database.driverClassName=org.postgresql.Driver
+          database.url=jdbc:postgresql://<strong>ip_of_postgres</strong>/<strong>database_name</strong>
+          database.username=postgres
+          database.password=postgres
+
+          vendorAdapter.database=POSTGRESQL
+        </source>
+        <p>If you are using the James Spring docker image. Create and edit james-database.properties file on local with above content then customize the configuration file of docker container throw volume</p>
+
+        <source>
+          -v path_to_your_local_folder_which_contain_configuration_file/james-database.properties:/root/james-server-app-3.0.0-RC2-SNAPSHOT/conf/james-database.properties
+        </source>
+      </subsection>
+
+      <subsection name="Step 3: Copy driver class to right place">
+        <p>Follow this link <a href="https://jdbc.postgresql.org/download.html">Postgres Driver</a> to download the right version of driver class </p>
+        <p>Copy that jar file to the lib folder of Spring James</p>
+        <p>If you are using the James Spring docker image. Download jar file then make it's available in the classpath of your James instance using an additional docker volume: </p>
+        <source>
+          -v path_to_your_local_folder_which_contain_jar_file/:/root/james-server-app-3.0.0-RC2-SNAPSHOT/conf/lib/
+        </source>
+      </subsection>
+
+      <subsection name="Step 4: Restart James">
+        <p>Follow the guide on <a href="http://james.apache.org/server/install.html">Restart and Testing</a> to restart James</p>
+
+        <p>If you are using the James Spring docker image. Just stop and start docker container again. </p>
```

Comment:
If you are using the James Spring docker image**, j**ust stop and start docker container again.